### ISSUE TITLE
More logging plugin support.

### ISSUE THREAD

**Issue by Magi1053** [Minebench/watson#3]
Is there, or will there be any support for Sponge block logging commands, such as Prism or AdamantineShield?
https://github.com/Karanum/AdamantineShield
https://github.com/prism/Prism

**Comment by Phoenix616** [comment]
Well the mod already has (experimental) support for Prism (did the Sponge version change the output format?) and it should not be too difficult to add support for more logging plugins. If you can provide some example log output messages then I could look into that.

Or you could open a PR yourself. The way to add support for new plugins should be pretty straight forward. See the [PrismAnalysis](https://github.com/Minebench/watson/blob/1.12.1/src/watson/analysis/PrismAnalysis.java) and [PrismPatterns](https://github.com/Minebench/watson/blob/1.12.1/src/watson/analysis/PrismPatterns.java) class.

**Comment by Magi1053** [comment]
Here is a screenshot of the AdamantineShield lookup
![image](https://user-images.githubusercontent.com/1045179/30094316-18e589f2-9291-11e7-93d3-9c5280d5b180.png)

**Comment by Phoenix616** [comment]
I tried adding an analyser for AdamantineShield. Please test out the [0.13.1-pre1 version](https://github.com/Minebench/watson/releases/tag/0.13.1-pre1-mc1.12.1) if it works as expected as I don't have a Sponge setup with which I could try it out.

**Comment by thisischrys** [comment]
Is AdamantineShield the only one on 1.12.2 supported right now? Or is there a list somewhere?

**Comment by Phoenix616** [comment]
> Is AdamantineShield the only one on 1.12.2 supported right now? Or is there a list somewhere?

No, the Mod should still support all of the plugins mentioned in the readme. It's focused on LogBlock though (which I use and can only really test) so if you notice any bugs with other plugins please report them.

**Comment by PixelmonPRO** [comment]
0.13.1-pre1 (and try add update to 1.12.2) version not work in 1.12.2 (does not display)
AdamantineShield v0.4.1 https://github.com/Karanum/AdamantineShield/releases/tag/0.4.1

**Comment by Phoenix616** [comment]
I don't really use that plugin so I can't really test it, I suggest taking a look at the [regex patterns](https://github.com/Minebench/watson/commit/3eb4a82d4bdd2afb19c6aee21a9e202df0981d84#diff-03fe082bf17d5e03f51ba2d626702571) and see if they match the plugin's messages.

**Comment by PixelmonPRO** [comment]
@Phoenix616 I tried to figure it out, but it didn't work out :(
Command `/as l`-> https://gyazo.com/1024efde13a39842748e2968ce368398
